control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -643,6 +639,23 @@ impl PublishOperation {
         self.publish_context.clone()
     }
 }
+/// SetVolumeProperty Operation parameters.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct VolumeProperty {
```

Comment:
Can we just directly use VolumeAttr everywhere?

Reply:
That's what I was wondering, just use the same (either Attr or Property). Or perhaps Property can be a type alias for Attr

Reply:
Fixed it. Before I have not used enum with datatype. With enum with datatype, this extra struct layering is not needed.